### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by anywherepilot**

### CODE REVIEW

File: src/commands/test/lib/wizard/uitest.ts [hunk 1]
```diff
@@ -39,7 +38,7 @@ export default class RunUitestWizardTestCommand extends AppCommand {
   }
 
   private async promptMergeXml(): Promise<boolean> {
-    const questions: Questions = [
+    const questions = [
```
Comment (jp-andre) [inline src/commands/test/lib/wizard/uitest.ts]: That's not a great change. Why remove the type here?

File: src/commands/codepush/deployment/lib/date-helper.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export function formatDate(unixOffset: number): string {
     if (differenceInMinutes(Date.now(), date) < 2) {
       formattedDateString = "Just now";
     } else {
-      formattedDateString = format(date, "MMM DD, hh:mm A");
+      formattedDateString = format(date, "MMM dd, hh:mm a");
```
Comment (jp-andre) [inline src/commands/codepush/deployment/lib/date-helper.ts]: How did you catch this, and is it tested?

File: src/commands/test/lib/wizard/uitest.ts (hunk 1 above)
Comment (anywherepilot) [reply src/commands/test/lib/wizard/uitest.ts]: The types are outdated. The `Questions` type no longer exist, and `Question` doesn't include `choices` so we can't make it a `Question[]` either.

Comment (anywherepilot) [reply src/commands/test/lib/wizard/uitest.ts]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inquirer/index.d.ts#L247

https://github.com/SBoudrias/Inquirer.js#questions

File: src/commands/codepush/deployment/lib/date-helper.ts (hunk 2 above)
Comment (anywherepilot) [reply src/commands/codepush/deployment/lib/date-helper.ts]: I tested it by emitting some dates somewhere else. I caught it by reading the doc: https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#200---2019-08-20

Comment (anywherepilot) [reply src/commands/codepush/deployment/lib/date-helper.ts]: The behavior of `MMM` also changed but I believe we still want to keep it.

**Comment by anywherepilot** [comment]
Not updating `gulp-typescript` because we're on the latest non-alpha release.

**Comment by anywherepilot** [comment]
Only `autorest` is left. I'm unable to run it on any of my machines either on `master` or after updating, so I'm reluctant to change it.
Node types shouldn't go up because we only support Node 10.

File: src/commands/codepush/deployment/lib/date-helper.ts (hunk 2 above)
Comment (jp-andre) [reply src/commands/codepush/deployment/lib/date-helper.ts]: Awesome, thanks for the link.

File: src/commands/test/lib/wizard/uitest.ts (hunk 1 above)
Comment (jp-andre) [reply src/commands/test/lib/wizard/uitest.ts]: So strange :/

File: test/integration/commands/help-test.ts [hunk 3]
```diff
@@ -3,10 +3,12 @@ import HelpCommand from "../../../src/commands/help";
 import { CommandArgs } from "../../../src/util/commandline";
 import { expect } from "chai";
 
-describe("Validating help command", () => {
+describe("Validating help command", function () {
 
   let stubbedConsole: Sinon.SinonStub;
 
+  this.timeout(5000);
```
Comment (jp-andre) [inline test/integration/commands/help-test.ts]: I slipped this in -- maybe it belongs to a separate PR?

Comment (anywherepilot) [reply test/integration/commands/help-test.ts]: Fine with me
